emerge-gitclone

```diff
@@ -51,6 +52,18 @@ def sync_repo():
     channel = get_channel()
     ref = f"{channel}-{release}"
 
+    # Check if release ends with a git ref. If it does we're dealing with
+    #   a dev build which does not necessarily have a tag on 'scripts'.
+    # These version strings look like e.g. "3552.0.0+nightly-20230323-2100-4-g5d72f4ee",
+    #   with "-g<checksum>" being a git ref.
+    gitref_pat=r'-g([0-9,a-f]+)$'
```

Comment:
I'd match upper-case letters too, just in case:
```suggestion
    gitref_pat=r'-g([0-9,a-f,A-F]+)$'
```
By the way - should really the ranges `0-9`, `a-f` and `A-F` be separated by a comma?

Reply:
No, they shouldn't - will remove. Wasn't aware git refs were upper case? Anyway, I'll add `A-F` and keep my fingers crossed `git checkout` will accept upper case :)

Reply:
Well, they are just hex numbers. And I just did `git checkout C4C40B54F76a983d43bbdc00e7377b17951df87a` (note: mixed case) on this repo using git 2.25.1 and it seemed to work.